### PR TITLE
Upgrade node6

### PR DESCRIPTION
**Pull request by pb82**
prepare for node6

### CODE REVIEW

File: fh-top/package.json [hunk 1]
```diff
@@ -5,24 +5,27 @@
     "feedhenry"
   ],
   "license": "Apache-2.0",
-  "version": "0.5.1",
+  "version": "0.5.2",
```
Comment (rachael-oregan) [inline fh-top/package.json]: Should this be a minor version bump?

File: fh-top/sonar-project.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
 sonar.projectKey=fh-top
 sonar.projectName=fh-top-nightly_master
-sonar.projectVersion=0.5.1
+sonar.projectVersion=0.5.2
```
Comment (rachael-oregan) [inline fh-top/sonar-project.properties]: Same here

File: sonar-project.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
 sonar.projectKey=fh-stats
 sonar.projectName=fh-stats-nightly_master
-sonar.projectVersion=0.1.0
+sonar.projectVersion=0.1.1
```
Comment (rachael-oregan) [inline sonar-project.properties:3]: And here?

File: fh-statsd/sonar-project.properties [hunk 4]
```diff
@@ -1,3 +1,3 @@
 sonar.projectKey=fh-statsd
 sonar.projectName=fh-statsd-nightly_master
-sonar.projectVersion=2.0.4
+sonar.projectVersion=2.0.5
```
Comment (rachael-oregan) [inline fh-statsd/sonar-project.properties]: Here too?

File: sonar-project.properties (hunk 3 above)
Comment (pb82) [reply sonar-project.properties:3]: Not sure about this one, both modules statsd and fh-top have their own sonar-project.properties

**Review by rachael-oregan — APPROVED** [review]

**Pull request merged**